base/MAPL_sun_uc.F90

```diff
@@ -2994,7 +2994,8 @@ subroutine MAPL_SunGetDaylightDurationMax(ORBIT,LATS,DAYL,currTime,CLOCK,RC)
          ! obliquity fixed in this case
          OBQ = ORBIT%OB * (MAPL_PI/180.)
       endif
-      _ASSERT(0. <= OBQ .and. OBQ < MAPL_PI, 'Strange obliquity!')
+      _ASSERT(OBQ >= 0, 'Obliquity less than 0 detected!')
```

Comment:
Not going to ask for changes here, but we really need to develop a consistent policy about how to word such messages.  And then _gradually_ propagate.